go.mod

```diff
@@ -1,5 +1,5 @@
-module "gopkg.in/yaml.v3"
+module gopkg.in/yaml.v3
 
-require (
-	"gopkg.in/check.v1" v0.0.0-20161208181325-20d25e280405
-)
+go 1.15
```

Comment:
Are we going to force the maintainers to update to go 1.15?

Reply:
![image](https://user-images.githubusercontent.com/49886065/114447775-9c4b4180-9b87-11eb-8665-1b61a9ffa2a0.png)

Reply:
and it came back grr. Deleted again